moveit_msgs/CollisionObject collObject
---